pipeline/pipeline_running/generate_release.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+ROOT_DIR=$(realpath $1)
+CREDENTIALS_PATH=$(realpath $2)
+PREVIOUS_RELEASE_DIR=$(realpath $3)
+
+GIT_COMMIT=${4-:"master"}
```

Comment:
I believe this should be GIT_COMMIT=${4:-"master"}

Reply:
yes, good catch!